src/applications/third-party-app-directory/components/SearchResult/index.unit.spec.js

```diff
@@ -0,0 +1,48 @@
+// Node modules.
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+// Relative imports.
+import { SearchResult } from './index';
+
+describe('<SearchResult>', () => {
+  it('should render', () => {
+    const props = {
+      item: {
+        name: 'Apple Health',
+        iconURL:
+          'https://www.apple.com/v/ios/health/c/images/overview/icon_health__c9yeoina5qaa_large_2x.png',
+        categories: ['Health'],
+        platforms: ['iOS'],
+        appURL: 'https://www.apple.com/ios/health/',
+        description:
+          'With the Apple Health app, you can see all your health records — such as medications, immunizations, lab results, and more — in one place. The Health app continually updates these records giving you access to a single, integrated snapshot of your health profile whenever you want, quickly and privately. All Health Records data is encrypted and protected with the user’s iPhone passcode, Touch ID or Face ID.',
+        permissions: [
```

Comment:
Permission scopes are being grabbed from the `/services/apps/v0/directory/scopes/{category_name}` endpoint now